src/traces/scattergl/style.js

```diff
@@ -0,0 +1,54 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Lib = require('../../lib');
+var Color = require('../../components/color');
+
+var DESELECTDIM = require('../../constants/interactions').DESELECTDIM;
+
+function styleTextSelection(cd) {
```

Comment:
Hmm, yeah this one isn't ideal. It is under a `style.js`, but it's not really a `_module.style` method. Ok if we move this thing to `./helpers.js` instead?